code/go/0chain.net/blobber/worker.go

```diff
@@ -38,8 +37,22 @@ func keepAliveOnChain() {
 
 	for {
 		time.Sleep(REPEAT_DELAY * time.Second)
-		if err := healthCheckOnChain(); err != nil {
+		err := handler.HealthCheckOnChain()
+		handler.SetBlobberHealthError(err)
+		if err != nil {
+			continue // pass // required by linting
+		}
+	}
+}
+
+func collectDBStats() {
+	const REPEAT_DELAY_SECONDS = 60 // 1 minutes
+
+	for {
+		err := handler.SetDBStats()
+		if err != nil {
 			continue // pass // required by linting
```

Comment:
it seems is incorrect. if it is `continue`, SetDbStats will be executed instantly. but it had `err`.

Reply:
Set DBStast() is not in working order (and will not be used), but it actually helped me to see that workers do not run since the addition of `isIntegrationTest`. Maybe you can see if the workers run for you. If you can run the workers then I can verify it using this code block. I will remove this code before we sqaush and merge.